src/shared/firewall-util.h

```diff
@@ -30,3 +31,33 @@ int fw_add_local_dnat(
                 const union in_addr_union *remote,
                 uint16_t remote_port,
                 const union in_addr_union *previous_remote);
+
+struct NFTSetContext {
```

Comment:
NFTSetContext vs NFTSetContexts is confusing. How about to name `NFTSet` and `NFTSetContext`?

Reply:
The difference is that NFTSetContext is one item and NFTSetContexts is an array of the same items. What about NFTSetContext and NFTSetContextArray?

Reply:
That sound too long for me. `NFTSet` for single item and `NFTSetContext` for array sounds sufficient for me.

But feel free to pick names you'd like.

Reply:
I used your suggestion.